playbooks/delete_resources.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
This file needs to move to playbooks/tasks/ so it's not treated as a playbook

Reply:
Added suggested change